www/ifwiki-check

```diff
@@ -13,21 +13,11 @@ curl_exec($ch);
 $status_code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
 curl_close($ch);
 
-$exists = ($status_code == 200);
-
+header("HTTP/1.1 $status_code");
```

Comment:
I prefer to use [`http_response_code()`](https://www.php.net/manual/en/function.http-response-code.php). (It's fine, either way.)

Reply:
Please change it, because I won't remember in the future without good examples to look at.

Reply:
(I copied this from another place in the code)